packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -130,15 +131,15 @@ abstract class FlutterGoldenFileComparator extends GoldenFileComparator {
   /// Calculate the appropriate basedir for the current test context.
   @protected
   @visibleForTesting
-  static Directory getBaseDirectory(LocalFileComparator defaultComparator, Platform platform) {
+  static Directory getBaseDirectory(LocalFileComparator defaultComparator, Platform platform, {bool unique = false}) {
```

Comment:
Maybe just allow for passing in a suffix and generate the (random) suffix where this is called?

Reply:
Done!